.github/workflows/ci.yml

```diff
@@ -170,10 +170,13 @@ jobs:
         choco install nuget.commandline
         nuget add -source local nuget/ParquetSharp.${{ steps.get-version.outputs.version }}.nupkg
     - name: Add local NuGet feed
-      run: dotnet nuget add source -n local $PWD/local
+      run: |
+        dotnet nuget add source -n local $PWD/local
+        dotnet nuget remove source nuget.org
```

Comment:
Could use a comment, esp. why are we removing nuget.org and then adding it back again? Did you get this from a stackoverflow page? Maybe link it?

Reply:
Good point, I have added a comment about it, though I'm not entirely sure it's actually doing anything (see https://github.com/NuGet/Home/issues/3676 for some context).